doc/en/components/charts/chart-features-trendlines.md

```diff
@@ -7,7 +7,87 @@ mentionedTypes: []
 
 # $PlatformShort$ Chart Trendlines
 
-<!--
+The $PlatformShort$ financial stock chart has support for trendlines, which help to identify a trend in data bound to a series.
 
-data-chart-series-trendlines.md
--->
\ No newline at end of file
+<code-view style="height: 500px" 
+           data-demos-base-url="{environment:dvDemosBaseUrl}" 
+           iframe-src="{environment:dvDemosBaseUrl}/charts/data-chart-series-trendlines" 
+           alt="$PlatformShort$ Trendlines Example" 
+           github-src="charts/data-chart/series-trendlines">
```

Comment:
why CV is pointing to DataChart samples and not FinancialChart sample?

Reply:
pending changes